assets/src/edit-story/app/media/media3p/useProviderContextValueProvider.js

```diff
@@ -59,6 +63,14 @@ export default function useProviderContextValueProvider(
     fetchMediaError,
   });
 
+  useFetchCategoriesEffect({
```

Comment:
What happens for providers that don't support categories, is useFetchCategoriesEffect a no-op to avoid the error in listCategories?

Reply:
We will need to implement that then accordingly.

Reply:
Do you want to handle that here, or if you want I can create a ticket in the Coverr epic?

Reply:
Let's tackle that in the Coverr epic.

Reply:
Done.
https://github.com/google/web-stories-wp/issues/3395